ParisBNF/abb/BNFabb114.xml

```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           woman who kisses his hand; second, the bishop sitting and cutting off his
+                           hand, and third, the bishop and the faithful standing in front of the icon of
+                           the <ref type="authFile" corresp="AT1031VirginAndChild"/>, which has
+                           healed his hand. 
+                        </desc>
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d8">
+                        <locus target="#20r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                           corresp="AT1113MMYeshaq"/> typically with the sequence of two episodes
+                           showing: first, the monk Yǝsḥaq falling on the ground during the
+                           apparition of Mary, and second, the icon of <persName ref="PRS6819Mary"/>
+                           taking Yǝsḥaq by the hand and raising him.
+                        </desc>
+                     </decoNote>
+
+                     <decoNote type="miniature" xml:id="d9">
+                        <locus target="#22r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                           corresp="AT1114MMMaryam"/>typically with sequence of one episode
+                           showing: <persName ref="PRS6819Mary"/> standing next to Māryām of Dafrā
+                           where the house represented with Jerusalem is depicted near to her.
+                        </desc>
+                     </decoNote>
+
+                     <decoNote type="miniature" xml:id="d10">
+                        <locus target="#24v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                           corresp="AT1115MMFrenchman"/> typically with sequence of three
+                           episodes/scenes, showing: first, the painter sitting on a scaffold and
+                           drawing a scene; second, the painter falling down from the scaffold and
+                           third, the icon of <persName ref="PRS6819Mary"/> extending her hand to saving
+                           the painter. 
+                        </desc> 
+                     </decoNote>
+
+                     <decoNote type="miniature" xml:id="d11">
+                        <locus from="25vb" to="26r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1116MMJerusalem"/> typically in four episodes/scenes,
+                           showing from the bottom left to the right: first, the sick pilgrim lying
+                           in bed; second the sick pilgrim standing before the places to which <persName ref="PRS6819Mary"/>
+                           carries him; third, in varying sequence Golgotha, Jerusalem, the mount of
```

Comment:
the same is true for "varying" as for "typically": This only makes sense in the art theme, here, there must be one concrete sequence